core/templates/pages/exploration-editor-page/services/non-strict-validation-fail-modal.controller.spec.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for NonStrictValidationFailModalController.
+ */
+
+import { WindowRef } from 'services/contextual/window-ref.service';
+
+describe('Non Strict Validation Fail Modal Controller', function() {
```

Comment:
Can we use the ES6 way of declaring functions and variables in this file?

Reply:
Done.